schema/notifications.graphql

```diff
@@ -221,9 +191,6 @@ type NftPurchased @variant {
 
   "price paid"
   price: BigInt!
-
-  "recipient"
-  recipient: RecipientType!
 }
 
 type CreatorReceivesAuctionBid @variant {
```

Comment:
I can't find the mapping for this one

Reply:
in `processNftBoughtEvent`, figma notification: NFT sold by seller